<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QWidget" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1025</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1024</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>DigiDoc4 Client</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QLabel {
color: #07142A;
}
#logo, #navBar {
background-color: #003168;
border-right: 1px solid #E7EAEF;
}
#topBar, #startScreen, #warnings {
background-color: #FFFFFF;
}
#warnings {
border-top: 1px solid #E7EAEF;
}
#signPage, #cryptoPage, #noReaderInfo {
background-image: url(:/images/bg_lovid.svg);
background-repeat: repeat-x;
background-position: bottom;
}
#signIntroButton, #cryptoIntroButton {
padding: 14px 26px;
border-radius: 4px;
font-family: Roboto, Helvetica;
font-size: 16px;
font-weight: 700;
color: #ffffff;
background-color: #2F70B6;
}
#signIntroButton:hover, #cryptoIntroButton:hover {
background-color: #2B66A6;
}
#signIntroButton:pressed, #cryptoIntroButton:pressed {
background-color: #215081;
}
#signIntroLabel, #cryptoIntroLabel, #noReaderInfoText {
font-family: Roboto, Helvetica;
font-size: 18px;
font-weight: 400;
}</string>
  </property>
  <layout class="QGridLayout" name="MainWindowLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <property name="spacing">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <widget class="QWidget" name="logo" native="true">
     <widget class="QSvgWidget" name="coatOfArms" native="true">
      <property name="geometry">
       <rect>
        <x>5</x>
        <y>11</y>
        <width>80</width>
        <height>32</height>
       </rect>
      </property>
      <property name="focusPolicy">
       <enum>Qt::TabFocus</enum>
      </property>
      <property name="accessibleName">
       <string notr="true">DigiDoc</string>
      </property>
     </widget>
    </widget>
   </item>
   <item row="1" column="0" rowspan="2">
    <widget class="QWidget" name="navBar" native="true">
     <property name="minimumSize">
      <size>
       <width>92</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>92</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">QToolButton {
border: none;
color: #E7EAEF;
padding: 13px 0px 10px 0px;
font-family: Roboto, Helvetica;
font-size: 16px;
}
QToolButton:checked {
background-color: #ffffff;
color: #003168;
}
#version, #version:pressed {
border: none;
color: #BFD3E8;
font-family: Roboto, Helvetica;
font-size: 10px;
text-decoration: underline;
}</string>
     </property>
     <layout class="QVBoxLayout" name="navBarLayout">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="PageIcon" name="signature">
        <property name="minimumSize">
         <size>
          <width>91</width>
          <height>93</height>
         </size>
        </property>
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="accessibleName">
         <string extracomment="accessible">Select signature page</string>
        </property>
        <property name="text">
         <string>Signature</string>
        </property>
        <property name="icon">
         <iconset resource="images/images.qrc">
          <normaloff>:/images/icon_Allkiri.svg</normaloff>
          <activeon>:/images/icon_Allkiri_hover.svg</activeon>:/images/icon_Allkiri.svg</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>40</width>
          <height>40</height>
         </size>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
        <property name="toolButtonStyle">
         <enum>Qt::ToolButtonTextUnderIcon</enum>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">pageButtonGroup</string>
        </attribute>
       </widget>
      </item>
      <item>
       <widget class="PageIcon" name="crypto">
        <property name="minimumSize">
         <size>
          <width>91</width>
          <height>93</height>
         </size>
        </property>
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="accessibleName">
         <string extracomment="accessible">Select crypto page</string>
        </property>
        <property name="text">
         <string>Crypto</string>
        </property>
        <property name="icon">
         <iconset resource="images/images.qrc">
          <normaloff>:/images/icon_Krypto.svg</normaloff>
          <activeon>:/images/icon_Krypto_hover.svg</activeon>:/images/icon_Krypto.svg</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>40</width>
          <height>40</height>
         </size>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <property name="toolButtonStyle">
         <enum>Qt::ToolButtonTextUnderIcon</enum>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">pageButtonGroup</string>
        </attribute>
       </widget>
      </item>
      <item>
       <widget class="PageIcon" name="myEid">
        <property name="minimumSize">
         <size>
          <width>91</width>
          <height>93</height>
         </size>
        </property>
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="accessibleName">
         <string extracomment="accessible">Select my EID page</string>
        </property>
        <property name="text">
         <string>My eID</string>
        </property>
        <property name="icon">
         <iconset resource="images/images.qrc">
          <normaloff>:/images/icon_Minu_eID.svg</normaloff>
          <activeon>:/images/icon_Minu_eID_hover.svg</activeon>:/images/icon_Minu_eID.svg</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>40</width>
          <height>40</height>
         </size>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <property name="toolButtonStyle">
         <enum>Qt::ToolButtonTextUnderIcon</enum>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">pageButtonGroup</string>
        </attribute>
       </widget>
      </item>
      <item>
       <spacer name="navBarSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="version">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>32</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>32</height>
         </size>
        </property>
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="text">
         <string notr="true">version</string>
        </property>
        <property name="flat">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QWidget" name="topBar" native="true">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>54</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>54</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">#help, #settings {
padding: 9px 8px;
border-radius: 4px;
color: #2F70B6;
font-family: Roboto, Helvetica;
font-size: 12px;
font-weight: 700;
}
#help:hover, #settings:hover {
background-color: #EAF1F8;
}
#help:pressed, #settings:pressed {
background-color: #BFD3E8;
}</string>
     </property>
     <layout class="QHBoxLayout" name="topBarLayout">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>6</number>
      </property>
      <property name="rightMargin">
       <number>16</number>
      </property>
      <property name="bottomMargin">
       <number>6</number>
      </property>
      <item>
       <widget class="CardWidget" name="cardInfo" native="true"/>
      </item>
      <item>
       <widget class="QToolButton" name="selector">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>32</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>32</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">border: none;</string>
        </property>
        <property name="icon">
         <iconset resource="images/images.qrc">
          <normaloff>:/images/arrow_down.svg</normaloff>
          <activeon>:/images/arrow_down_selected.svg</activeon>:/images/arrow_down.svg</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>24</width>
          <height>24</height>
         </size>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="NoCardInfo" name="noCardInfo" native="true"/>
      </item>
      <item>
       <spacer name="topBarSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>8</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <layout class="QHBoxLayout" name="tbButtonWidgetLayout">
        <property name="spacing">
         <number>8</number>
        </property>
        <item>
         <widget class="QToolButton" name="help">
          <property name="cursor">
           <cursorShape>PointingHandCursor</cursorShape>
          </property>
          <property name="layoutDirection">
           <enum>Qt::RightToLeft</enum>
          </property>
          <property name="text">
           <string>Help</string>
          </property>
          <property name="icon">
           <iconset>
            <normalon>:/images/icon_Abi.svg</normalon>
           </iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>14</width>
            <height>14</height>
           </size>
          </property>
          <property name="toolButtonStyle">
           <enum>Qt::ToolButtonTextBesideIcon</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="settings">
          <property name="cursor">
           <cursorShape>PointingHandCursor</cursorShape>
          </property>
          <property name="text">
           <string>Settings</string>
          </property>
          <property name="icon">
           <iconset>
            <normalon>:/images/icon_Seaded.svg</normalon>
           </iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>14</width>
            <height>14</height>
           </size>
          </property>
          <property name="toolButtonStyle">
           <enum>Qt::ToolButtonTextBesideIcon</enum>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="WarningList" name="warnings" native="true">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>1</height>
      </size>
     </property>
     <layout class="QVBoxLayout" name="warningsLayout">
      <property name="spacing">
       <number>2</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>1</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
     </layout>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QStackedWidget" name="startScreen">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="signPage">
      <layout class="QVBoxLayout" name="signPageLayout">
       <property name="spacing">
        <number>40</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <spacer name="signPageTopSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>159</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLabel" name="signIntroLabel">
         <property name="text">
          <string>Drag file here for signing ...</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item alignment="Qt::AlignHCenter">
        <widget class="QPushButton" name="signIntroButton">
         <property name="cursor">
          <cursorShape>PointingHandCursor</cursorShape>
         </property>
         <property name="accessibleName">
          <string comment="accessible">Load file from disk for signing or verifying</string>
         </property>
         <property name="text">
          <string>... or load file from disk</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="signPageBottomSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>260</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="ContainerPage" name="signContainerPage"/>
     <widget class="QWidget" name="cryptoPage">
      <layout class="QVBoxLayout" name="cryptoPageLayout">
       <property name="spacing">
        <number>40</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <spacer name="cryptoPageTopSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>159</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLabel" name="cryptoIntroLabel">
         <property name="text">
          <string>Drag file here for encryption ...</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item alignment="Qt::AlignHCenter">
        <widget class="QPushButton" name="cryptoIntroButton">
         <property name="cursor">
          <cursorShape>PointingHandCursor</cursorShape>
         </property>
         <property name="accessibleName">
          <string comment="accessible">Load file from disk for encryption or decryption</string>
         </property>
         <property name="text">
          <string>... or load file from disk</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="cryptoPageBottomSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>260</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="ContainerPage" name="cryptoContainerPage"/>
     <widget class="QWidget" name="eidPage">
      <layout class="QVBoxLayout" name="eidPageLayout">
       <property name="spacing">
        <number>3</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="InfoStack" name="infoStack" native="true">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>172</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>186</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="Accordion" name="accordion" native="true"/>
       </item>
       <item>
        <widget class="QWidget" name="noReaderInfo" native="true">
         <layout class="QVBoxLayout" name="noReaderInfoLayout">
          <property name="spacing">
           <number>40</number>
          </property>
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <spacer name="noReaderInfoTopSpacer">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>159</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QLabel" name="noReaderInfoText">
            <property name="focusPolicy">
             <enum>Qt::TabFocus</enum>
            </property>
            <property name="text">
             <string>Connect the card reader to your computer and insert your ID card into the reader</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
            <property name="wordWrap">
             <bool>true</bool>
            </property>
            <property name="openExternalLinks">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QWidget" name="noReaderInfoTextSpacer" native="true">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>42</height>
             </size>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="noReaderInfoBottomSpacer">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>260</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QSvgWidget</class>
   <extends>QWidget</extends>
   <header location="global">QSvgWidget</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>CardWidget</class>
   <extends>QWidget</extends>
   <header>widgets/CardWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ContainerPage</class>
   <extends>QWidget</extends>
   <header>widgets/ContainerPage.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>InfoStack</class>
   <extends>QWidget</extends>
   <header>widgets/InfoStack.h</header>
  </customwidget>
  <customwidget>
   <class>Accordion</class>
   <extends>QWidget</extends>
   <header>widgets/Accordion.h</header>
  </customwidget>
  <customwidget>
   <class>NoCardInfo</class>
   <extends>QWidget</extends>
   <header>widgets/NoCardInfo.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>PageIcon</class>
   <extends>QToolButton</extends>
   <header>widgets/PageIcon.h</header>
  </customwidget>
  <customwidget>
   <class>WarningList</class>
   <extends>QWidget</extends>
   <header>widgets/WarningList.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="images/images.qrc"/>
 </resources>
 <connections/>
 <buttongroups>
  <buttongroup name="pageButtonGroup"/>
 </buttongroups>
</ui>
